portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -150,4 +148,14 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * Sends an error to the client as raw text instead of the default HTML page.
+   */
+  private void sendRawTextError(HttpServletResponse response, int errorCode, String errorMsg) 
+    throws IOException {
```

Comment:
nit: I think I prefer if `throws IOException {` goes up on the previous line along with the rest of the method definition?

Reply:
I agree, however I reached the line length limit with the method declaration. Is there a better way I could format this or should I ignore the length limit?

Reply:
That's fine then! Just need 4 spaces instead of 2 https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent

Thanks